site/docs/main/configuration.md

```diff
@@ -106,6 +107,16 @@ The cluster configuration block can be used to configure various parameters for
 {: class="table thead-dark table-bordered"}
 <br>
 
+### Network Configuration
```

Comment:
Do we have any issues we can earmark so we make sure to add configuration to this object?

Reply:
There's an issue linked for the max hops, are you referring to the `Network Configuration` section?

Reply:
Yeah, just wondering if we already had other things to add in the `network` hash we already know about

Reply:
Yeah at the moment not sure, things that the chunked length maybe should have gone into there. We can remove the `network` parent, but was trying to group stuff together in a way.

Reply:
I think having the `network` top-level item is great, we should definitely keep that, and look out for things to put in there.

Reply:
👍